include/gatekeeper_cps.h

```diff
@@ -28,8 +28,9 @@
 
 extern int cps_logtype;
 
-#define CPS_LOG(level, ...) \
-	rte_log(RTE_LOG_ ## level, cps_logtype, "GATEKEEPER CPS: " __VA_ARGS__)
+#define CPS_LOG_RATELIMIT(level, ...)                    \
+	rte_log_ratelimit(RTE_LOG_ ## level, cps_logtype,\
```

Comment:
Why did you rename `CPS_LOG()` to `CPS_LOG_RATELIMIT()` instead of keeping the name? Wouldn't replacing `rte_log()` with `rte_log_ratelimit()` be enough?

Reply:
As we are ratelimiting the log entries, I added ``_RATELIMIT`` to reflect this fact. I am OK with keeping the name.